<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tiendas</class>
 <widget class="QDialog" name="tiendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiendas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imagenes/Emeicjac logo.jpg</normaloff>:/imagenes/Emeicjac logo.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="8">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNuevo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Nuevo</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBorrar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Borrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModificar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRefrescar">
       <property name="text">
        <string>Refrescar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnterior">
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSiguiente">
       <property name="text">
        <string>Siguiente</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3" colspan="8">
    <widget class="QWidget" name="general_2" native="true">
     <layout class="QGridLayout" name="general">
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Whatsapp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ususario</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEditIP">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ciudad</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>I.P.</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditWharsapp"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Direcci&amp;ón</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditTlfn"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditId">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Contrase&amp;ña</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEditUsusario"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Correo</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Teléfono</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditNombre"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditDireccion"/>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditCiudad"/>
      </item>
      <item row="7" column="3">
       <widget class="QCheckBox" name="checkBoxMaster">
        <property name="text">
         <string>¿Es la tienda principal?</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QCheckBox" name="checkBoxLocal">
        <property name="text">
         <string>¿Es la tienda local?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditNombre</tabstop>
  <tabstop>lineEditDireccion</tabstop>
  <tabstop>lineEditCiudad</tabstop>
  <tabstop>lineEditTlfn</tabstop>
  <tabstop>lineEditWharsapp</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditIP</tabstop>
  <tabstop>lineEditUsusario</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonNuevo</tabstop>
  <tabstop>pushButtonBorrar</tabstop>
  <tabstop>pushButtonModificar</tabstop>
  <tabstop>pushButtonRefrescar</tabstop>
  <tabstop>pushButtonAnterior</tabstop>
  <tabstop>pushButtonSiguiente</tabstop>
  <tabstop>pushButtonCerrar</tabstop>
  <tabstop>lineEditId</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCerrar</sender>
   <signal>clicked()</signal>
   <receiver>tiendas</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
